[0KRunning with gitlab-ci-multi-runner 9.5.1 (96b34cc)
  on Shared Gitlab Runner Token (87897d2f)
[0;m[0KUsing Docker executor with image centos:6 ...
[0;m[31;1mERROR: Preparation failed: Error response from daemon: error creating aufs mount to /var/lib/docker/aufs/mnt/5d411255e357e4077501552ab1805b223a83327bddd604792ac20b6e1baab90e-init: invalid argument
[0;m[32;1mWill be retried in 3s ...
[0;m[0KUsing Docker executor with image centos:6 ...
[0;m[31;1mERROR: Preparation failed: Error response from daemon: error creating aufs mount to /var/lib/docker/aufs/mnt/864beeda5833be0cdea28a77c26856011359fba709e649d2d0cea40a423e64ab-init: invalid argument
[0;m[32;1mWill be retried in 3s ...
[0;m[0KUsing Docker executor with image centos:6 ...
[0;m[31;1mERROR: Preparation failed: Error response from daemon: error creating aufs mount to /var/lib/docker/aufs/mnt/39f3d4db9cde609b9df223b61c57caa92ee47203d958246c7d053c05cd6d0956-init: invalid argument
[0;m[32;1mWill be retried in 3s ...
[0;m[31;1mERROR: Job failed (system failure): Error response from daemon: error creating aufs mount to /var/lib/docker/aufs/mnt/39f3d4db9cde609b9df223b61c57caa92ee47203d958246c7d053c05cd6d0956-init: invalid argument
[0;m